Makefile

```diff
@@ -44,4 +46,23 @@ deps:
 install:
 	for d in $(SUBDIRS); do $(MAKE) -C $$d install; done
 
-.PHONY: all $(SUBDIRS) clean proto deps lint install
+docker-images:
```

Comment:
Can we separate this out into multiple tasks? That way we can add a stub file for each one, and if someone wants to rebuild a specific image it isn't too much work.

Reply:
Yeah for sure, I split them into separate targets and made the `docker-images` target just run all of them (in order so that we maximize re-use of build cache since the different images are mostly layers on top of each other)